packages/rest-typings/src/v1/chat.ts

```diff
@@ -255,8 +255,9 @@ export const isChatReportMessageProps = ajv.compile<ChatReportMessage>(ChatRepor
 
 type ChatGetThreadsList = PaginatedRequest<{
 	rid: IRoom['_id'];
-	type: 'unread' | 'following' | 'all';
+	type?: 'unread' | 'following' | 'all';
```

Comment:
I didn't find any mention of `all` in the code... Is this expected?

Reply:
all is not handled specifically because it works just as when type is not sent (no extra filter is applied). Do you think we should remove this option?

Reply:
I think it's better to remove it, to avoid some confusion